src/App.js

```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import MyCalculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
```

Comment:
- [ ] For this project, it's required that the App.js component be class-based components. Kindly modify the stateless functional component to a class-based component 👍

Reply:
@AmaduKamara would you please comment on the current README.md and advise?...i believe its descriptive